Sources/Passes/Models/Concrete Models/PKPass.swift

```diff
@@ -11,7 +11,9 @@ import FluentKit
 /// The `Model` that stores PassKit passes.
 ///
 /// Uses a UUID so people can't easily guess pass serial numbers.
-open class PKPass: PassModel, @unchecked Sendable {
+final public class PKPass: PassModel, @unchecked Sendable {
```

Comment:
Too generic?
```suggestion
final public class Pass: PassModel, @unchecked Sendable {
```

Reply:
I left it like this because the file extension representing passes is .pkpass, as opposed to orders which is .order, but if you think it causes confusion I can change it

Reply:
I did change it, since we could add support for Google Wallet in future.